radicle-node/src/service.rs

```diff
@@ -595,11 +600,18 @@ where
                     Ok(updated) => {
                         log::debug!(target: "service", "Fetched {rid} from {remote}");
 
-                        self.reactor.event(Event::RefsFetched {
+                        let event = Event::RefsFetched {
                             remote,
                             rid,
                             updated: updated.clone(),
-                        });
+                        };
+                        self.reactor.event(event.clone());
```

Comment:
```suggestion
```

Reply:
what happens to the reactor then? are we removing it? or does reactor subscribe and then forwards the events?

Reply:
removed it [here](https://github.com/radicle-dev/heartwood/compare/151868ed3c0ec50054de690b24126c57aa5e247a..4a65b33f3ac878437c41bafce9fa3993e0e357ce), waiting for the substitute.

Reply:
I think [test](https://github.com/radicle-dev/heartwood/actions/runs/4416542559/jobs/7741336583) is failing because of this, need your input re this.

Reply:
I tried a few things just to experiment, such as putting reactor inside emitter, but couldn't get it to compile. I guess it's harder than it looks, maybe we should abandon reactor `Vec` model, and use `Receiver` in another thread?

Reply:
The reactor is not needed for events anymore, since you are sending them over another channel. We just need to read from that channel in the tests.